README.md

```diff
@@ -39,12 +32,45 @@ def application do
 end
 ```
 
+## If you'd like to mount VerkWeb on another Endpoint:
+
+
+```elixir
+defmodule MyApp.Endpoint do
+  use VerkWeb.Mount, path: "/verk"
+  ...
+end
+```
+
+That should be it! :)
```

Comment:
No, it's already false, see line 25 in mix.exs (https://github.com/edgurgel/verk_web/pull/15#discussion-diff-59111293R25) 

We only need to set it to true if launching as standalone app...